assets/sass/patterns/molecules/article-download-links-list.scss

```diff
@@ -76,22 +77,30 @@
     @include padding(24, "right");
     @include menu-typeg();
     text-decoration: none;
+    font-weight: normal;
   }
 
   .article-download-links-list__heading {
+    @include font-size(12);
     @include label-content-typeg();
     margin: 0;
     @include blg-spacing("top", "small");
-    padding-bottom: 0;
+    @include blg-spacing("bottom", "extra-small");
     @include padding(24, "left");
     @include padding(24, "right");
   }
 
   @media only all and (min-width: #{get-rem-from-px($bkpt-content-header--medium-width)}em) {
-    right: 16px;
+    right: 45px;
   }
 
   @media only all and (min-width: #{get-rem-from-px($bkpt-content-header--extra-wide)}em) {
     right: 0;
   }
 }
+
+.article-download-links-list__group--exclude {
```

Comment:
@discodavey why was it necessary to add this class could we not have targeted an existing class. Introduced here: https://github.com/elifesciences/pattern-library/commit/b8c8c0c5f09de1ab6173fc6a0d72e8e06cbbdcba

Reply:
It would have affected other sections in the page that did not need the changes. I did not want to use the class article-download-links-list__group--js-exclude as it's just for JS only.

Reply:
It's safe to just target `.article-download-links-list__intro` in all cases.